src/transformers/models/musicgen/modeling_musicgen.py

```diff
@@ -572,6 +572,24 @@ def forward(
                 output_attentions=output_attentions,
             )
 
+        # Ignore copy
+        if (
+            attention_mask is not None
+            and (attention_mask.mean(dim=[1, 2, 3]) <= torch.finfo(attention_mask.dtype).min).any()
```

Comment:
What's the reason for using finfo here and not just 0?

Reply:
The attention has already been processed and is filled with `-inf` of the correspondant dtype!